lib/models/game.rb

```diff
@@ -0,0 +1,24 @@
+require 'errors/game_over_error'
+
+module RubyConnect
+  class Game
+    attr_accessor :board, :winning_move
+
+    def initialize(player1, player2)
+      @board = Board.new
+      @winning_move = false
+      @player1 = player1
+      @player2 = player2
+    end
+
+    def done?
+      board.full? || winning_move
+    end
+
+    def make_move(column, color)
+      raise GameOverError if done?
```

Comment:
I thought about the case when a player make a move even the game is over. Maybe this is not an issue in a browser, cause you can disable input or something like that. But, via SMS, I think this can make sense. What do you think about it?